ironfist/src/cpp/scripting/scripting.h

```diff
@@ -4,16 +4,32 @@
 #include <map>
 #include <list>
 
-enum MapVarType {MAPVAR_TYPE_STRING, MAPVAR_TYPE_NUMBER, MAPVAR_TYPE_TABLE, MAPVAR_TYPE_BOOLEAN, MAPVAR_TYPE_ERROR};
+enum MapVarType {
+  MAPVAR_TYPE_STRING,
+  MAPVAR_TYPE_NUMBER,
+  MAPVAR_TYPE_TABLE,
+  MAPVAR_TYPE_BOOLEAN,
+  MAPVAR_TYPE_ERROR
+};
 
 typedef std::map<std::string, std::pair<MapVarType, std::string>> luaTableElements;
 typedef std::list<std::string> luaSubTables;
 typedef std::map<std::string, std::pair<luaTableElements, luaSubTables>> luaTable;

```

Comment:
@rems75 

Remind me why this current representation of tables. I don't find this intuitive.